source/install/verify-signatures/macos.txt

```diff
@@ -0,0 +1,142 @@
+.. _verify-signatures-macos:
+
+======================================
+Verify MongoDB Shell Packages on macOS
+======================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. include:: /includes/verify-signatures-intro.rst
+
+Before you Begin
+----------------
+
+If you don't have MongoDB Shell installed, download the MongoDB Shell
+binary from the `Download Center
+<https://www.mongodb.com/try/download/shell?jmp=docs>`__.
+
+Steps
+-----
+
+To verify MongoDB Shell packages on macOS, you can use one of these
+methods:
+
+- :ref:`Disk Image Verification <verify-signatures-macos-disk-image>`
+
+- :ref:`GPG <verify-signatures-macos-gpg>`
+
+.. _verify-signatures-macos-disk-image:
+
+Verify Packages with Disk Image Verification
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run the following command:
+
+.. code-block:: sh
+
+   codesign -dv --verbose=4 <path_to_mongosh>
```

Comment:
Let's make it clear that `path_to_mongosh` is the path to the executable, not the `.zip` file?

Reply:
Updated